itou/templates/apply/submit_base.html

```diff
@@ -10,19 +10,15 @@ <h1>{% include 'apply/includes/_submit_title.html' %}</h1>
     <section class="s-section">
         <div class="s-section__container container">
             <div class="row">
-                <div class="col-12 col-lg-8">
-                    {% if is_subject_to_eligibility_rules %}
-                        {% with common_approval=job_seeker.latest_common_approval %}
-                            {% if common_approval %}
-                                <div class="c-box mb-4">
-                                    {% include "approvals/includes/status.html" with common_approval=common_approval hiring_pending=True %}
-                                </div>
-                            {% endif %}
-                        {% endwith %}
-                    {% endif %}
-
+                <div class="col-12 col-lg-8 {% if is_subject_to_eligibility_rules and job_seeker.has_valid_approval %}col-lg-8 order-2 order-lg-1{% endif %}">
```

Comment:
Classe CSS en double :
```suggestion
                <div class="col-12 col-lg-8 {% if is_subject_to_eligibility_rules and job_seeker.has_valid_approval %}order-2 order-lg-1{% endif %}">
```